web/src/components/App.tsx

```diff
@@ -66,13 +84,36 @@ class App extends React.Component<Props> {
     this.props.history.push(`/${this.props.match.params.service}/errors/${encodeURIComponent(errorKey)}`)
   }
 
+  handleFilterByAggregatedkey = (key: string) => {
+    const { errors } = this.props
+
+    switch (errors.status) {
+      case RemoteData.SUCCESS:
+
+        return this.setState({
+          errors: errors.data.filter((error) => error.aggregation_key.includes(key.toLowerCase())),
```

Comment:
@sikozonpc thanks a lot for the change, it looks great!

I noticed a small bug though: I think we need to to call `toLowerCase()` also for `error.aggregation_key`, otherwise the search term won't mach the exception when the exception key contains upper case letters. You can test it with an exception key containing upper case letters (`com.test.CancelledRequestException`) and searching for a term containing the upper case part.

Cheers and happy to see new contributions from you!

Reply:
Hey  @jcreixell , thats for the kind words and the feedback.
Yhea small mistake there indeed, I'll adress it in a new PR.